### DISCUSSION TITLE
Status of Rhino

### DISCUSSION THREAD

**Discussion by wshallwshall** [mozilla/rhino#1176] [Q&A] [answered]
Can you tell me the current status of this project? Is the Mozilla Foundation still funding this project?
If not, is all the work done by volunteers?

**Comment by p-bakker** [comment, marked as answer]
As mentioned in the [readme](https://github.com/mozilla/rhino#contributing-prs) the Rhino project is an all-volunteer effort.

May I ask why you're asking?
